python/ray/data/read_api.py

```diff
@@ -540,6 +541,67 @@ def read_mongo(
     )
 
 
+@PublicAPI(stability="alpha")
+def read_bigquery(
+    project_id: str,
+    dataset: Optional[str] = None,
+    query: Optional[str] = None,
+    *,
+    parallelism: int = -1,
+    ray_remote_args: Dict[str, Any] = None,
+    **bigquery_args,
+) -> Dataset:
+    """Create an Arrow dataset from BigQuery.
+
+    The data to read from is specified via the ``project_id``, ``dataset``
+    and/or ``query``parameters. The dataset is created from the results of
+    executing``query`` if a query is provided. Otherwise, the entire
+    ``dataset`` will be read.
+
+    You can check out more details here about these BigQuery concepts:
+    - Project id: https://cloud.google.com/resource-manager/docs/creating-managing-projects # noqa: E501
+    - Dataset: https://cloud.google.com/bigquery/docs/datasets-intro # noqa: E501
+    - Query: https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax # noqa: E501
```

Comment:
Could you link these and use the corresponding document titles as the link text?

Reply:
Done